libraries/Microsoft.Bot.Schema/GeoCoordinates.cs

```diff
@@ -73,17 +73,10 @@ public GeoCoordinates(double? elevation = default(double?), double? latitude = d
         [JsonProperty(PropertyName = "longitude")]
         public double? Longitude { get; set; }
 
-        /// <summary>
-        /// Gets or sets the type of the thing
-        /// </summary>
-        [JsonProperty(PropertyName = "type")]
-        public string Type { get; set; }
-
```

Comment:
@tomlm The field is still in the class, this does not remove the field. It removes the, seemingly unintentional, hiding of the based class field with the same type and name.

When this field is here then, depending on how you access the field. you get different values for it. People code around it by de-serializing multiple times... trying each type it could be until one works. If this field goes away that code would still work.

Reply:
Tagging @mrivera-ms  also.

Reply:
Currently people call this

https://github.com/microsoft/botbuilder-dotnet/blob/d085ac8220bbd990565db75639fa53d9816bf6bf/libraries/Microsoft.Bot.Schema/EntityEx.cs#L31

Like:
```
var geoCoordinates = entity.GetAs<GeoCoordinates>();
if (geoCoordinates.Type == nameof(GeoCoordinates)) {}
var type2 = entity.GetAs<Type2>();
if (type2.Type == nameof(Type2)) { }
etc 
etc
```

Each GetAs deserializes